src/domain/help/pages/registrationInstructions/RegistrationInstructions.tsx

```diff
@@ -0,0 +1,311 @@
+            </p>
+
+            {getImage(1, 'en')}
+            <p>
+              Select the event for which you want to create a registration; you
+              can also type in the field.
+            </p>
+            <p>
+              Note! You cannot create a registration for an umbrella event, so
+              you will not find them in the menu.
+            </p>
+            {getImage(2, 'en')}
+            <p>
+              Once an event is selected, the system automatically imports the
+              registration time, seat and queue information and age limits for
+              the event.
+            </p>
+            <p>
+              If necessary, specify the maximum size of the group. This
+              determines how many participants can register on one form. If no
+              value is entered here, the number of available places in the event
+              determines the number of people who can register on a single form.
+              If it is not desired that more than one person can be registered
+              per form, this will be set to 1.
+            </p>
+            <p>
+              Select the input languages you wish to use and enter any
+              registration instructions and additional text for the confirmation
+              message in the languages you have chosen. Please note that the
+              languages displayed above the input fields for registration
+              instructions and confirmation messages are tab options.
+            </p>
+            {getImage(3, 'en')}
+            <p>
+              Please select the mandatory personal data fields on the
+              registration form.
+            </p>
+            <p>
+              <strong>
+                Note! In accordance with the EU Data Protection Regulation,
+                personal data should not be collected unnecessarily, so please
+                exercise discretion as to which personal data are necessary for
+                registration.
+              </strong>
+            </p>
+            {getImage(4, 'en')}
+            <p>
+              Finally, add read access rights to the participant list for people
+              who need to access the participant list.
+            </p>
+            <p>Save your registration.</p>
+
+            <h2>Enrolment management</h2>
+            <p>
+              On the front page of the registration section, you will see a list
+              of events for which registration has been created.
+            </p>
+            {getImage(5, 'en')}
+            <p>
+              From the front page, you can create a new registration by clicking
+              on the Add new button.
+            </p>
+            <p>
+              You can also use the three-dot menu at the end of the events bar:
+            </p>
+            <ul>
+              <li>edit already created enrolment</li>
+              <li>View the list of participants</li>
```

Comment:
Should all of these begin with a capital letter?

Reply:
All `li` elements are now capitalized.